packages/structure/src/model/RWRoute.ts

```diff
@@ -144,6 +154,11 @@ export class RWRoute extends BaseNode {
       )
     if (this.hasPathCollision)
       yield err(this.path_literal_node!, 'Duplicate Path')
+    if (this.hasInvalidPrivateRoute)
+      yield err(
+        this.jsxNode!,
+        "The 'Not Found' page cannot be within a <Private> tag"
+      )
     if (this.isAuthenticated && this.isNotFound)
```

Comment:
Should references to `isAuthenticated` be removed? In https://github.com/redwoodjs/redwood/issues/917 there's a comment by @aldonline indicating that references to `authenticated` are no longer being used in favor of `private`.

> You'll notice that there is some old code/stub that references a property called route.isAuthenticated. The "isAuthenticated" name is from before the "Private" tag name was chosen.

Reply:
Yeah. We need to clean that up. It's a good thing that you didn't try to do that in this same PR, since it would touch two separate parts of the codebase.